src/main/java/com/wavesplatform/wavesj/Node.java

```diff
@@ -299,6 +306,25 @@ public OrderStatusInfo getOrderStatus(String orderId, AssetPair assetPair) throw
         return parse(r, ORDER_LIST);
     }
 
+    public Map<String, Long> getTradableBalance(AssetPair pair, String address) throws IOException {
+        String path = String.format("/matcher/orderbook/%s/%s/tradableBalance/%s", pair.getAmountAsset(), pair.getPriceAsset(), address);
+        HttpResponse r = exec(request(path));
+        return parse(r, RESERVED);
+    }
+
+    private Map<String, Long> getReserved(PrivateKeyAccount account, String path) throws IOException {
```

Comment:
Why do you need this function? It's only used in one place

Reply:
yes, it's useless. done